lighthouse-core/test/fraggle-rock/api-test-pptr.js

```diff
@@ -119,10 +123,18 @@ describe('Fraggle Rock API', () => {
       const bestPractices = lhr.categories['best-practices'];
       expect(bestPractices.score).toBeLessThan(1);
 
-      const {auditResults, erroredAudits, failedAudits} = getAuditsBreakdown(lhr);
+      const {
+        auditResults,
+        erroredAudits,
+        failedAudits,
+        notApplicableAudits,
+      } = getAuditsBreakdown(lhr);
       // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
       expect(auditResults.length).toMatchInlineSnapshot(`63`);
 
+      expect(notApplicableAudits).toHaveLength(8);
```

Comment:
should we inline snapshot this? I expect this number to change

Reply:
ya